drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -169,6 +169,10 @@ def head_of_household?
     self.RelationshipToHoH == 1
   end
 
+  def age_over_18?
+    client.age.present? && client.age > 18
+  end
```

Comment:
It's really tempting to put `def adult?` and `def child?` on Client.  what do you think?

Reply:
Child < 18, Adult > 18.  Youth depends on context, sometimes 18-24, sometimes 12-24, sometimes < 24.

Reply:
I was just thinking those might be useful methods, though they should probably take a date.

Reply:
I added `child?` and `adult?` with optional dates to the shared Client concern

Reply:
Thank you!